### PR TITLE
Casmnet 921

### PR DESCRIPTION
**Pull request by coreyjjohnson**
## Summary and Scope

Update verbiage for manual port configuration on Storage node ports for both Aruba and Mellanox. 
Updated code examples taken from internal system, with ports that don't overlap.

## Issues and Related PRs

* Resolves [CASMNET-921](https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-921)
* Documentation changes required in [CASMNET-921](https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-921)
* Merge with/before/after `<release/1.0>`

## Testing

Doc change. Fixing accuracy

### Tested on:

  * `<HELA>`
  * Local development environment
  * Virtual Shasta

### Test description:

Just a doc change.

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/network/management_network/Management_Network_Access_Port_Configurations.md [hunk 1]
```diff
@@ -106,14 +106,14 @@ interface lag 17 multi-chassis
 ## Gigabyte/Intel NCN Worker Port Configuration
 The cabling guidelines for all servers can be found in [Cable Management Network Servers](../../../install/cable_management_network_servers.md).
 
-Mellanox port configuration:
+Aruba port configuration:
```
Comment (trad511) [inline operations/network/management_network/Management_Network_Access_Port_Configurations.md:109]: Customers should never have an Aruba combined with Gigabyte/Intel.

**Review by trad511 — CHANGES_REQUESTED** [review]
@coreyjjohnson Please fill out the PR text, replacing the boilerplate text.  Without this it's very difficult to tell what this PR is about without digging in to changes or the Jira.

Also please make sure the Gigabyte/Intel example uses Mellanox, not Aruba.

File: operations/network/management_network/Management_Network_Access_Port_Configurations.md [hunk 2]
```diff
@@ -106,14 +106,14 @@ interface lag 17 multi-chassis
...
 ```
 sw-spine-002 [gamora-mlag-domain: master] # show run int ethernet 1/1
 interface ethernet 1/1 speed 40G force
 interface ethernet 1/1 mtu 9216 force
 interface ethernet 1/1 mlag-channel-group 1 mode active
 ```
-Mellanox MLAG configuration:
+Aruba LAG configuration:
```
[the remark this answers is not in the document]
Comment (coreyjjohnson) [reply operations/network/management_network/Management_Network_Access_Port_Configurations.md]: Updated PR summary.

Corrected and committed text change from Aruba to Mellanox.

File: operations/network/management_network/Management_Network_Access_Port_Configurations.md (hunk 1 above)
Comment (coreyjjohnson) [reply operations/network/management_network/Management_Network_Access_Port_Configurations.md]: Correccted in db050a52

**Review by trad511 — APPROVED** [review]

**Pull request merged**